packages/@dcl/inspector/src/components/Renderer/Renderer.tsx

```diff
@@ -10,14 +10,15 @@ import { useSdk } from '../../hooks/sdk/useSdk'
 import { getPointerCoords } from '../../lib/babylon/decentraland/mouse-utils'
 import { snapPosition } from '../../lib/babylon/decentraland/snap-manager'
 import { ROOT } from '../../lib/sdk/tree'
-import { changeSelectedEntity } from '../../lib/utils/gizmo'
 import { AssetNodeItem } from '../ProjectAssetExplorer/types'
 import { IAsset } from '../AssetsCatalog/types'
 import { getModel, isAsset } from '../EntityInspector/GltfInspector/utils'
 
 import './Renderer.css'
 import { useIsMounted } from '../../hooks/useIsMounted'
 
+const fixedNumber = (val: number) => Math.round(val * 1e2) / 1e2
```

Comment:
```suggestion
const fixedNumber = (val: number) => val.toFixed(2)
```

Reply:
I dont know why I did this, i think it was to avoid casting to Nubmer again, too many iterations 😂